java/pom.xml

```diff
@@ -87,6 +87,31 @@
           <autoReleaseAfterClose>true</autoReleaseAfterClose>
         </configuration>
       </plugin>
+      <plugin>
```

Comment:
Formatting source code using this Spotless plugin.
for https://github.com/google/open-location-code/pull/325/files#r281754908
You just need to run the spotless apply from plugins to format the source code. 
No human effort involved anymore.